src/NHibernate/Type/ManyToOneType.cs

```diff
@@ -19,20 +19,37 @@ public partial class ManyToOneType : EntityType
 		public ManyToOneType(string className)
 			: this(className, false)
 		{
+			
 		}
 
 		public ManyToOneType(string className, bool lazy)
 			: base(className, null, !lazy, false)
+		{
+			ignoreNotFound = false;
+			isLogicalOneToOne = false;
+			PropertyName = null;
+		}
+		
+		public ManyToOneType(string className, string propertyName, bool lazy)
```

Comment:
Is this ctor used? `propertyName` param is not used

Reply:
I forgot to remove this ctor, sorry